aws/resource_aws_storagegateway_smb_file_share_test.go

```diff
@@ -844,3 +919,53 @@ resource "aws_storagegateway_smb_file_share" "test" {
 }
 `, tagKey1, tagValue1, tagKey2, tagValue2)
 }
+
+func testAccAWSStorageGatewaySmbFileShareSMBACLConfig(rName string, enabled bool) string {
+	return testAccAWSStorageGateway_SmbFileShare_ActiveDirectoryBase(rName) + fmt.Sprintf(`
+resource "aws_storagegateway_smb_file_share" "test" {
+  authentication  = "ActiveDirectory"
+  gateway_arn     = "${aws_storagegateway_gateway.test.arn}"
```

Comment:
🔜  0.11 syntax CI failures 😂  Not 🔜  enough. (I tease! I know you have a huge backlog of open things that we need to get through)